src/plugins/gauge/GaugeCompositionPolicy.js

```diff
@@ -0,0 +1,29 @@
+export default function GaugeCompositionPolicy(openmct) {
+  function hasNumericTelemetry(domainObject) {
+    const hasTelemetry = openmct.telemetry.isTelemetryObject(domainObject);
+    if (!hasTelemetry) {
+      return false;
+    }
+
+    let metadata = openmct.telemetry.getMetadata(domainObject);
```

Comment:
`const` ?

Reply:
Yep; copy and paste from OverlayPlotCompositionPolicy.js. Have changed in the Gauge version.